doc/plugins/lineprofiler.rst

```diff
@@ -0,0 +1,52 @@
+####################
+Spyder Line Profiler
+####################
+
+**Spyder-Line-Profiler** is a plugin to run the Python `line_profiler <https://pypi.python.org/pypi/line_profiler>`_.
+
+.. image:: /images/lineprofiler/lineprofiler-overview.png
+   :alt: Spyder with the Line Profiler pane open
+
+The line_profiler will profile the time that individual lines of code take to execute.
+
+============================
+Installing the Line Profiler
+============================
+
+If you installed Spyder using conda, the best way to install Spyder-terminal is to run the following command in your terminal or Anaconda prompt on Windows:
+
+.. code-block:: bash
+
+   conda install spyder-line-profiler -c spyder-ide
+
+.. important::
+
+   At the moment it is not possible to use this plugin with the Spyder :ref:`standalone_installers_ref` for Windows and macOS. We're working to make that possible in the future.
+
+Restart Spyder in order to be able to use the plugin.
+
+=======================
+Using the Line Profiler
+=======================
+
+When the Line Profiler is installed, it will be available under the menu item :menuselection:`View --> Panes --> Line Profiler`.
+
+.. image:: /images/lineprofiler/lineprofiler-view-panes.png
+   :alt: Spyder showing View Panes Line Profiler
+
+You will see it then as a tab next to the 'Files' tab. You can then either select a script with the button present in the pane or run the profiler from :menuselection:`Run --> Profile line by line`.
+your script line by line.
```

Comment:
```suggestion
Afterwards, your file will be profiled line by line.
```

- Give a better start to this sentence.
- Avoid repeating `your script`, which was mentioned in the previous sentence.
- Mention again what the plugin does (just for clarity).